ateam_ai/src/ateam_ai_node.cpp

```diff
@@ -144,6 +147,19 @@ class ATeamAINode : public rclcpp::Node
     // Preproccess world
     //
     world_.current_time += 0.01;
+    for (int i = 0; i < 16; i++) {
```

Comment:
It seems like in other places we are iterating through robot ids we use `std::size_t robot_id` as the variable which is more obvious what is happening